src/Homework4/task7.py

```diff
@@ -0,0 +1,17 @@
+""" Даны два натуральных числа.
+ Вычислите их наибольший общий делитель при помощи
+алгоритма Евклида (мы не знаем функции и рекурсию)
+"""
+
+
+a = 12
+b = 18
+
+while a != 0 and b != 0:
+    if a > b:
+        a = a % b
+
+    else:
+        b = b % a
+
+print("Наибольший общий делитель чисел", a + b)
```

Comment:
It's a cheat - you just don't know where is result (second var is 0).
There is solution without this crutch